iceoryx_hoofs/test/moduletests/test_logstream.cpp

```diff
@@ -265,24 +272,95 @@ void testStreamOperatorLogBin(Logger_Mock& loggerMock, LogType logValue)
                 Eq("0b" + std::bitset<std::numeric_limits<TestType>::digits>(outputValue).to_string()));
 }
 
-TYPED_TEST(IoxLogStreamHexBin_test, StreamOperatorLogBin_ValueLow)
+TYPED_TEST(IoxLogStreamHexOctBinIntegral_test, StreamOperatorLogBin_ValueLow)
 {
     ::testing::Test::RecordProperty("TEST_ID", "e4e684b7-5bcf-4e8d-8cb1-b6df95c3b37c");
     testStreamOperatorLogBin(this->loggerMock, this->LogValueLow);
 }
 
-TYPED_TEST(IoxLogStreamHexBin_test, StreamOperatorLogBin_ValueMin)
+TYPED_TEST(IoxLogStreamHexOctBinIntegral_test, StreamOperatorLogBin_ValueMin)
 {
     ::testing::Test::RecordProperty("TEST_ID", "f13b3e6a-8f7c-48c2-ae43-35e0a195556e");
     testStreamOperatorLogBin(this->loggerMock, this->LogValueMin);
 }
 
-TYPED_TEST(IoxLogStreamHexBin_test, StreamOperatorLogBin_ValueMax)
+TYPED_TEST(IoxLogStreamHexOctBinIntegral_test, StreamOperatorLogBin_ValueMax)
 {
     ::testing::Test::RecordProperty("TEST_ID", "b583014a-700f-46e3-8b7c-0a128c59598a");
     testStreamOperatorLogBin(this->loggerMock, this->LogValueMax);
 }
 
+#endif
+
+template <class T>
+class IoxLogStreamHexFloatingPoint_test : public IoxLogStream_test
+{
+  public:
+    T LogValueLow = std::numeric_limits<T>::lowest();
+    T LogValueMin = std::numeric_limits<T>::min();
+    T LogValueMax = std::numeric_limits<T>::max();
+};
+
+using LogHexFloatingPointTypes = Types<float, double, long double>;
+
+#ifdef __clang__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wgnu-zero-variadic-macro-arguments"
+#endif
+TYPED_TEST_SUITE(IoxLogStreamHexFloatingPoint_test, LogHexFloatingPointTypes);
```

Comment:
As before, you can remove the warning by adding an additional comma.

Reply:
see https://github.com/eclipse-iceoryx/iceoryx/pull/1621#discussion_r974558928